go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/bacalhau-project/bacalhau
 
 go 1.20
 
+//quic-go is tied to a specific go version: look at https://github.com/quic-go/quic-go/wiki/quic-go-and-Go-versions
```

Comment:
I don't think we need this comment here, we're aware of the issue

Reply:
ok